rate-providers/GTokenRateProvider.md

```diff
@@ -2,7 +2,7 @@
 
 ## Details
 - Reviewed by: @mkflow27
-- Checked by: @\<GitHub handle of secondary reviewer\>
+- Checked by: @danielmkm
```

Comment:
@danielmkm does this mean you have reviewed it? if so, can you then do a formal review of the pr, and possible merge it?

we use this repo downstream for reviewing payloads, but we only consider `main` branch

Reply:
hey @gosuto-inzasheru just in the process of wrapping it up now!

Reply:
sorry, i didnt realise your commit was so fresh when i opened this pr :) thanks a lot